### PR TITLE
Create imagemagick_policy.xml

### PR DESCRIPTION
**Pull request by briankassouf**

### CODE REVIEW

**Comment by lospro7** [comment]
:+1:

**Comment by kodermike** [comment]
Did you confirm there was nothing in the existing file already that this would wipe? The default policy file comes with other policies you may not want to destroy that are outside the scope of this issue.

**Comment by kodermike** [comment]
https://git.imgur-dev.com/Imgur/Configs/pull/394/files

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM golang:1.5
 RUN apt-get update && apt-get install -yqq aspell aspell-en libaspell-dev tesseract-ocr tesseract-ocr-eng imagemagick optipng exiftool libjpeg-progs webp
 ADD docker/meme.traineddata /usr/share/tesseract-ocr/tessdata/meme.traineddata
+ADD docker/imagemagick_policy.xml /etc/ImageMagick-6/policy.xml
```
Comment (lospro7) [inline Dockerfile]: it's not `/etc/ImageMagick` like the blog post suggests?

**Comment by kodermike** [comment]
:+1:

**Pull request merged**